Control/DeepSeq.hs

```diff
@@ -3,6 +3,9 @@
 #if __GLASGOW_HASKELL__ >= 702
```

Comment:
Well, formally I'm the maintainer of `deepseq`, c.f. https://wiki.haskell.org/Library_submissions#The_Libraries

That being said, if the CLC objects I'd listen :-)